Dom/last.js

```diff
@@ -0,0 +1,127 @@
+function dropFunction() {
+	document.getElementById("myDrop").classList.toggle("show");
+}
+
+// Close the dropdown menu if the user clicks outside of it
+window.onclick = function(event) {
+	if (!event.target.matches('.dropbtn')) {
+
+		var dropdowns = document.getElementsByClassName("dropdown-content");
+		var i;
+		for (i = 0; i < dropdowns.length; i++) {
```

Comment:
do u really need a for loop here?
this can be done by removing show class of the dropdown-container